scout/commands/load/base.py

```diff
@@ -37,4 +38,5 @@ def load():
 load.add_command(delivery_report_command)
 load.add_command(cnv_report_command)
 load.add_command(coverage_qc_report_command)
+load.add_command(gene_fusion_report_command)
```

Comment:
We probably want a load_custom_report at some point (major =), but this is perfect now!

Reply:
I totally agree, it would be so nice to generalize and allow people to provide and visualize any report they want. We could already start thinking about it